apps/www/components/landing/WaitlistForm.tsx

```
@@ -1,4 +1,29 @@
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+  const enroll = trpc.waitlist.enroll.useMutation();
+
+  const handleSubmit = async () => {
+    if (!email) {
+      setMessage("Email is required");
+      return;
+    }
+    try {
+      const result = await enroll.mutateAsync({ email });
+      if (result && result.enrollment_id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)